admin/update

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -eu
+
+#
+# Update firmware.
+#
+
+# prevent modification of executable path
+PATH="/bin:/sbin:/usr/bin:/usr/sbin"
+
+# parse command line options
+FILE=
+RESET=
+YES=
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -h | --help )
+      echo "Update BIOS/BMC firmware."
+      echo "Use: ${0##*/} [OPTION...] FILE"
+      echo "  -r, --reset   Reset all settings to manufacturing default"
```

Comment:
Please consult the CLI proposal document on intranet Confluence

Reply:
Fixed.